Chore: remove last few v2 directories

froddd opened the pull request:
Follow-up to https://github.com/ministryofjustice/hmpps-temporary-accommodation-ui/pull/1456 with a few more directories cleaned up.

Mostly straight file moves, but some combination going on with integration/e2e test pages in `cypress_shared/pages/temporary-accommodation/manage`, where the v2 test methods have been added to the original non-v2 pages. Those methods have been lifted verbatim with no functional changes at all, other than where needed (matching difering signatures, types, etc). The integration tests themselves have not had to be changed beyond imports.

Import/include/extend paths in views have been shortened to the absolute relative to the views root for the files moved, as this is considered a default views include and makes managing these paths a bit easier.

froddd committed:
Remove more v2 directories for integration tests

froddd committed:
Remove more v2 directories in cypress shared

froddd committed:
Remove last few v2 playwright pages

froddd committed:
Move v2 views alongside existing

Update all import paths to views root, as this makes managing these
tempaltes easier.

froddd commented on cypress_shared/pages/temporary-accommodation/manage/bedspaceEditable.ts:
```diff
@@ -1,18 +1,18 @@
-    if (this.bedStartDateIsEditable() && newOrUpdateBedspace.endDate) {
-      this.getLegend('Enter the bedspace end date (optional)')
-      this.completeDateInputs('startDate', newOrUpdateBedspace.endDate)
+    if (this.bedStartDateIsEditable() && newOrUpdateBedspace.startDate) {
+      this.getLegend('Enter the bedspace start date')
+      this.completeDateInputs('startDate', newOrUpdateBedspace.startDate)
```
I _think_ this is correct -- the original code didn't make a lot of sense, but do let me know if that's right!

SaintRhys commented on server/views/temporary-accommodation/bedspaces/edit.njk:
```diff
@@ -31,19 +31,20 @@
-                            text: 'Bedspace reference',
-                            classes: "govuk-label--m"
-                        },
+                        text: 'Bedspace reference',
+                        classes: "govuk-label--m"
+                    },
                         classes: "govuk-input--width-20",
                         hint: {
-                            text: "This will be used to identify the bedspace when making bookings and reporting on the service."
-                        },
+                        text: "This will be used to identify the bedspace when making bookings and reporting on the service."
+                    },
```
Super small, might be an autoformatting thing but these indentations look off compared to the rest of the file/s, another in `_editable.njk` and there's a few more, with `html: dateHtml`

froddd replied:
Ah yes, that is the auto formatting -- I believe the best way to avoid this is to ensure the helper opening is on one line, i.e.:

```nunjucks
{{ formPageInput({
  label: ''
  // ...
}) }}
```

instead of:
```nunjucks
{{ formPageInput(
  {
    label: '',
    // ...
  }
) }}
```

AvraamK replied:
Awesome! This has been bugging me for some time now!

SaintRhys reviewed (commented):
Just a little one on formatting, feel free to ignore

froddd committed:
Fix Cypress E2E test issues

froddd committed:
Improve Cypress local config

This change prevents retries locally, but ensures DOM snapshots are
available for a number of tests to help with local debugging.

AvraamK commented on cypress.config.e2e.ts:
```diff
@@ -51,6 +51,6 @@ export default defineConfig({
   },
-  numTestsKeptInMemory: 0,
-  retries: 1,
+  numTestsKeptInMemory: process.env.CI ? 0 : 20,
```
nice!

froddd replied:
This means running E2E tests locally will be a bit faster when they fail, but it also enables clicking around on each step of the test to inspect values, DOM, etc. It was becoming absolutely essential working on these!
